src/commands/deploy.ts

```diff
@@ -47,7 +47,29 @@ export const TARGET_PERMISSIONS: Record<(typeof VALID_DEPLOY_TARGETS)[number], s
     "firebaserules.releases.update",
   ],
   remoteconfig: ["cloudconfig.configs.get", "cloudconfig.configs.update"],
-  dataconnect: [], // TODO: add required permissions
+  dataconnect: [
```

Comment:
Is this just supposed to be the smallest set of permissions that people will need to use the CLI?

Reply:
Yes (or at least, specifically, the `firebase deploy` command). I know some of these won't be used in all cases (ie you won't need `cloudsql.instances.create` if you are using an existing instance) - I'd like to come back and refine this to support those cases later